### PR TITLE
[ES] FAQ - Added "Controls" and "Change Gamemode" Sections

### PR DESCRIPTION
**Pull request by Mikes-Play**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [X] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/FAQ/es.md [hunk 1]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
 
+### Controles
+
+Los controles predeterminados para Osu! son:
+
+| Mouse | Teclado | Tableta |
+| :-- | :-- | :-- |
+| Click izquierdo(M1) / Click derecho(M2) | `Z`(K1) / `X`(K2) | Tocar la pantalla/tableta(M1) |
+
+Los Hit Objects en Osu! aceptarán cualquier entrada proveniente de cualquier dispositivo siempre y cuando los Hit Objects sean presionados en el momento correcto
```
Comment (agatemosu) [inline wiki/FAQ/es.md:102]:
```suggestion
Los objetos en osu! aceptarán cualquier entrada proveniente del dispositivo y cuando los objetos sean presionados en el momento correcto.
```

File: wiki/FAQ/es.md [hunk 2]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+Si se está usando el modificador [Relax](/wiki/es/Gameplay/Game_modifier/Relax), solo funcionará el cursor y se deshabilitarán los botones del teclado/mouse/lapiz. Usa el cursor para seguir a los Hit Objects mientras estos se presionan automaticamente. Sin embargo, los Spinners deberan ser completados normalmente.
```
Comment (agatemosu) [inline wiki/FAQ/es.md:104]:
```suggestion
Si se está usando el modificador del juego [Relax](/wiki/Gameplay/Game_modifier/Relax), solo funcionará el cursor y se deshabilitarán los botones del teclado/ratón/lápiz. Usa el cursor para seguir a los objetos mientras estos se presionan automaticamente. Sin embargo, los spinners deberán ser completados normalmente.
```

File: wiki/FAQ/es.md [hunk 3]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+Si se está usando el modificador [Auto Pilot](/wiki/es/Gameplay/Game_modifier/Autopilot), solo funcionará la entrada de teclado/mouse/lapiz pero se deshabilitará el movimiento del cursor. Usa los botones para presionar a tiempo los Hit Objects mientras el cursor los sigue automaticamente. Los Spinners seguirán la velocidad del mod [Spun Out](/wiki/es/Gameplay/Game_modifier/Spun_Out)
```
Comment (agatemosu) [inline wiki/FAQ/es.md:106]:
```suggestion
Si se está usando el modificador del juego [Auto Pilot](/wiki/es/Gameplay/Game_modifier/Autopilot), solo funcionará la entrada del teclado/ratón/lápiz pero se deshabilitará el movimiento del cursor. Usa los botones para presionar a tiempo los objetos mientras el cursor los sigue automaticamente. Los spinners seguirán la velocidad del mod [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out)
```

File: wiki/FAQ/es.md [hunk 4]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+---
+
 ## Otros Modos de Juego
 
+### Cambiando el Modo de Juego
```
Comment (agatemosu) [inline wiki/FAQ/es.md:112]:
```suggestion
## Otros modos de juego

### Cambiando el modo de juego
```

File: wiki/FAQ/es.md [hunk 5]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+![Mode change interface](/wiki/shared/Interface_mode.png "Mode toolbox in Song Selection (Solo)")
```
Comment (agatemosu) [inline wiki/FAQ/es.md:114]:
```suggestion
![Interfaz del cambio de modo](/wiki/shared/Interface_mode.png "Cambio de modo en el selector de canciones (Solo)")
```

File: wiki/FAQ/es.md [hunk 6]
```diff
@@ -91,8 +91,30 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+Cliquea en el boton de `Modo` para **abrir una lista con los modos de juego disponibles en Osu!**. Cliquea en el modo de juego deseado y Osu! cambiará al estilo del modo de juego deseado. Tambien puedes presionar las teclas `Ctrl` y `1` (![][osu!] *[osu!](/wiki/es/Game_mode/osu!))* / `2` ( ![][osu!taiko] *[osu!taiko](/wiki/es/Game_mode/osu!taiko)*) / `3` (![][osu!catch] *[osu!catch](/wiki/es/Game_mode/osu!catch)*) / `4` (![][osu!mania] *[osu!mania](/wiki/es/Game_mode/osu!mania)*) para cambiar rapidamente entre los modos de juego
```
Comment (agatemosu) [inline wiki/FAQ/es.md:116]:
```suggestion
Haz clic en el boton de `Mode` para **abrir una lista con los modos de juego disponibles en osu!**. Haz clic en el modo de juego deseado y osu! cambiará al modo de juego deseado. También puedes presionar las teclas `Ctrl` y `1` (![][osu!] *[osu!](/wiki/Game_mode/osu!))* / `2` ( ![][osu!taiko] *[osu!taiko](/wiki/Game_mode/osu!taiko)*) / `3` (![][osu!catch] *[osu!catch](/wiki/Game_mode/osu!catch)*) / `4` (![][osu!mania] *[osu!mania](/wiki/Game_mode/osu!mania)*) para cambiar rápidamente entre los modos de juego
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/FAQ/es.md [hunk 7]
```diff
@@ -91,7 +91,29 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
 - El número en la parte inferior izquierda es el contador de combo/multiplicador de puntaje.
 
-## Otros Modos de Juego
+### Controles
+
+Los controles predeterminados para osu! son:
+
+| Ratón | Teclado | Tableta |
+| :-- | :-- | :-- |
+| Clic izquierdo(M1) / Clic derecho(M2) | `Z`(K1) / `X`(K2) | Tocar la pantalla/tableta(M1) |
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/FAQ/es.md:100]: así sale en el original

[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/FAQ/es.md:100]: Ya, pero se ve más bonito así

File: wiki/FAQ/es.md [hunk 8]
```diff
@@ -91,7 +91,29 @@ Esta es la interfaz del estilo de juego osu! Standard.
...
+
+Si se está usando el modificador del juego [Auto Pilot](/wiki/es/Gameplay/Game_modifier/Autopilot), solo funcionará la entrada del teclado/ratón/lápiz pero se deshabilitará el movimiento del cursor. Usa los botones para presionar a tiempo los objetos mientras el cursor los sigue automaticamente. Los spinners seguirán la velocidad del mod [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out)
+
+---
+
+## Otros modos de juego
+
+### Cómo cambiar el modo de juego
+
+![Interfaz para cambiar el modo de juego](/wiki/shared/Interface_mode.png "Menú desplegable para la selección de los modos de juego en el selector de canciones (Solo)")
```
Comment (agatemosu) [inline wiki/FAQ/es.md:114]:
```suggestion
![Menú de cambio de modo en el selector de canciones (Solo)](/wiki/shared/Interface_mode.png "Menú de cambio de modo en el selector de canciones (Solo)")
```

**Review by agatemosu — APPROVED** [review]

**Review by UnknownOsu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**